src/MagnumPlugins/BasisImageConverter/BasisImageConverter.conf

```diff
@@ -77,6 +77,10 @@ rdo_uastc_favor_simpler_modes_in_rdo_mode=true
 ktx2_uastc_supercompression=true
 ktx2_zstd_supercompression_level=6
 
+# OpenCL acceleration. Falls back to CPU encoding if OpenCL isn't supported or
+# fails during encoding.
+use_opencl=false
```

Comment:
This default makes sense given that multithreading is off by default as well, but I'm thinking I'll eventually make it multithreaded + with OpenCL by default, since it makes sense to have things fast without having to set plugin-specific opt-in values.